<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>729</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="setRegAddrButton">
       <property name="text">
        <string>Set Reg Addr</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="readRegAddrButton">
       <property name="text">
        <string>Read Reg Addr</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="powerOffButton">
       <property name="text">
        <string>Power Off</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>621</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>671</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>MIDR</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>CTR</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TCMTR</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TLBTR</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>MPIDR</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>REVIDR</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>ID_PFR0</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ID_PFR1</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>ID_DFR0</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>ID_AFR0</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>ID_MMFR0</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>ID_MMFR1</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>ID_MMFR2</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>ID_MMFR3</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>ID_ISAR0</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>ID_ISAR1</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>ID_ISAR2</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>ID_ISAR3</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>ID_ISAR4</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>ID_ISAR5</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>CCSIDR</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>CLIDR</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>AIDR</string>
       </property>
      </widget>
     </item>
     <item row="23" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>CSSELR</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="midr_le"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ctr_le"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="tcmtr_le"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="tlbtr_le"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="mpidr_le"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="revidr_le"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="id_pfr0_le"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="id_pfr1_le"/>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="id_dfr0_le"/>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="id_afr0_le"/>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="id_mmfr0_le"/>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="id_mmfr1_le"/>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="id_mmfr2_le"/>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="id_mmfr3_le"/>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="id_isar0_le"/>
     </item>
     <item row="15" column="1">
      <widget class="QLineEdit" name="id_isar1_le"/>
     </item>
     <item row="16" column="1">
      <widget class="QLineEdit" name="id_isar2_le"/>
     </item>
     <item row="17" column="1">
      <widget class="QLineEdit" name="id_isar3_le"/>
     </item>
     <item row="18" column="1">
      <widget class="QLineEdit" name="id_isar4_le"/>
     </item>
     <item row="19" column="1">
      <widget class="QLineEdit" name="id_isar5_le"/>
     </item>
     <item row="20" column="1">
      <widget class="QLineEdit" name="ccsidr_le"/>
     </item>
     <item row="21" column="1">
      <widget class="QLineEdit" name="clidr_le"/>
     </item>
     <item row="22" column="1">
      <widget class="QLineEdit" name="aidr_le"/>
     </item>
     <item row="23" column="1">
      <widget class="QLineEdit" name="csselr_le"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
